pontoon/base/urls.py

```diff
@@ -38,6 +38,10 @@
         views.translate,
         name='pontoon.translate'),
 
+    url(r'^(?P<locale>[A-Za-z0-9\-]+)/(?P<slug>[\w-]+)/(?P<filename>[A-Za-z0-9\-\.]+)\.tmx$',
```

Comment:
Let's keep the regexes for `locale`, `slug` and (as a consequence of the two) `filename` consistent with other URL patterns, e.g. `pontoon.translate`.